code/style.css

```diff
@@ -2,6 +2,307 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+
+*{
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	list-style: none;
+	font-family: 'Montserrat', sans-serif;
+}
+
+/* =================
+header
+==================== */
+body, html {
+  height: 100%;
+}
+
+h1 {
+
+font-family: 'Rock Salt', cursive;
+}
+
+h2 {
+font-family: 'Comforter', cursive;
+font-size: 30px;
+}
+
+.hero-image {
+height: 50vh;
+background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("./hero-img.jpg");
+height: 50%;
+background-position: center;
+background-repeat: no-repeat;
+background-size: cover;
+position: relative;
+}
+
+.hero-text {
+text-align: center;
+position: absolute;
+top: 50%;
+left: 50%;
+transform: translate(-50%, -50%);
+color: white;
+}
+
+.section{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+section {
+  padding: 6rem, 2rem;
+	background: #D3DEDC;
+}
+
+/* =================
+sign up
+==================== */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+
+.registration_form{
+	background: #4B6587;
+	padding: 30px;
+	border-radius: 5px;
+	width: 400px;
+  height: 457px;
+  align-items: center;
+  margin-right: 20px;
+  margin-top: 40px;
+  margin-bottom: 40px;
```

Comment:
If you want to, you can write all margins in one line, something like this: 

margin: 40px 20px 40px 0px;

It goes like a clock, so the first thing you write is the top-margin, second is the right-margin and so on. I recently learned this and it has helped me alot!